logger/logger.go

```diff
@@ -14,7 +16,7 @@ func InitLogger() *zerolog.Logger {
 	consoleWriter := zerolog.ConsoleWriter{Out: os.Stdout}
 	multiWriter := zerolog.MultiLevelWriter(consoleWriter, logFile)
 
-	logger := zerolog.New(multiWriter).With().Timestamp().Logger()
-
-	return &logger
+	log := zerolog.New(multiWriter).With().Timestamp().Logger()
+	Log = &log
+	return &log
```

Comment:
```suggestion
	log.Logger = zerolog.New(multiWriter).With().Timestamp().Logger()
```

can be reduced to this